python/pyarrow/tests/test_parquet.py

```diff
@@ -3345,3 +3345,41 @@ def test_filter_before_validate_schema(tempdir):
     # read single file using filter
     table = pq.read_table(tempdir, filters=[[('A', '==', 0)]])
     assert table.column('B').equals(pa.chunked_array([[1, 2, 3]]))
+
+
+@pytest.mark.pandas
+@pytest.mark.fastparquet
+@pytest.mark.filterwarnings("ignore:RangeIndex:DeprecationWarning")
+def test_fastparquet_cross_compatibility(tempdir):
+    fp = pytest.importorskip('fastparquet')
+
+    df = pd.DataFrame(
+        {
+            "a": list("abc"),
+            "b": list(range(1, 4)),
+            "c": np.arange(4.0, 7.0, dtype="float64"),
+            "d": [True, False, True],
+            "e": pd.date_range("20130101", periods=3),
+            "f": pd.Categorical(["a", "b", "a"])
+            # "g": [[1, 2], None, [1, 2, 3]],
```

Comment:
I assume it's commented out because it currently doesn't work? Add a comment describing why?

Reply:
Yes, indeed, I was first trying it, but apparently fastparquet writes such column as BYTE_ARRAY JSON (while we write is as a actual nested INT64 column). 
I added a comment